### PR TITLE
linter fixes for fasttree

### PR DESCRIPTION
**Pull request by bernt-matthias**
Leaving the `InputsBoolDistinctValues` skip: the boolean has no `truevalue` and `falsevalue`. It's used in a `filter` and an `if` statement of the command block .. and this works. 

FOR CONTRIBUTOR:
* [ ] I have read the [CONTRIBUTING.md](https://github.com/galaxyproject/tools-iuc/blob/master/CONTRIBUTING.md) document and this tool is appropriate for the tools-iuc repo.
* [ ] License permits unrestricted use (educational + commercial)
* [ ] This PR adds a new tool or tool collection
* [ ] This PR updates an existing tool or tool collection
* [ ] This PR does something else (explain below)

### CODE REVIEW

File: tools/fasttree/fasttree.xml [hunk 1]
```diff
@@ -48,7 +48,7 @@ $model_selector.model
 #if $save_logfile:
     2> '$log'
 #end if
-
+&& echo $save_logfile
```
Comment (bgruening) [inline tools/fasttree/fasttree.xml:51]:
```suggestion
```

File: tools/fasttree/fasttree.xml [hunk 2]
```diff
@@ -1,12 +1,12 @@
 <?xml version="1.0"?>
 <tool id="fasttree" name="FASTTREE" version="@VERSION@+galaxy2" profile="18.01">
```
Comment (bgruening) [inline tools/fasttree/fasttree.xml:2]:
```suggestion
<tool id="fasttree" name="FASTTREE" version="@VERSION@+galaxy2" profile="23.1">
```

Not needed if you don't want to bump.

File: tools/fasttree/.lint_skip [hunk 3]
```diff
@@ -1,4 +1 @@
 InputsBoolDistinctValues
```
Comment (bernt-matthias) [inline tools/fasttree/.lint_skip:1]:
```suggestion
# the bool is used in a filter and an if statement of the command block which works
InputsBoolDistinctValues
```

**Review by bgruening — APPROVED** [review]

**Comment by mvdbeek** [comment]
Attention: deployment failure!

https://github.com/galaxyproject/tools-iuc/actions/runs/11192302077

**Comment by mvdbeek** [comment]
> Leaving the `InputsBoolDistinctValues` skip: the boolean has no `truevalue` and `falsevalue`. It's used in a `filter` and an `if` statement of the command block .. and this works.

How can this work ? https://github.com/mvdbeek/galaxy/blob/user_defined_tools/lib/galaxy/tool_util/parser/util.py#L72-L86
This should be a linting error on profile >= 23.1. The tool doesn't even parse, so no tests were run.

**Comment by bernt-matthias** [comment]
Do you see a possibility to make the test fail if the tool fails to load?

**Comment by mvdbeek** [comment]
Maybe, the parser lives in tool-util ? This is covered by the linter, we should just not disable it.

**Pull request merged**